### PR TITLE
[DOC] Rename OTX to OpenVINO™ Training Extensions

### PR DESCRIPTION
**Pull request by supersoob**
Please tell me if there's something wrong.

### CODE REVIEW

File: docs/source/guide/release_notes/index.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ v1.0.0 (1Q23)
 ***************
 
 - Installation through PyPI
-  - Package will be renamed as OTX (OpenVINO Training eXtensions)
+  - Package will be renamed as OpenVINO™ Training Extensions (OpenVINO Training eXtensions)
```
Comment (chuneuny-emily) [inline docs/source/guide/release_notes/index.rst]: Can you remove (OpenVINO Training eXtensions)?

File: docs/source/guide/get_started/introduction.rst [hunk 2]
```diff
@@ -7,9 +7,9 @@
...
+OpenVINO™ Training eXtensions (OpenVINO™ Training Extensions) is a command-line interface (CLI) framework designed for low-code computer vision deep-learning model training. OpenVINO™ Training Extensions lets users train, infer, optimize and deploy models simply and fast even with low expertise in the deep learning field. OpenVINO™ Training Extensions offers a diverse combination of model architectures, learning methods, and training types using PyTorch and OpenVINO™ toolkit. OpenVINO™ Training Extensions provides so-called "model templates" for every supported task which have been tested on various datasets and are a turnkey solution for obtaining an average good model without the need to change any hyperparameters. Besides, it is possible to configure your own model based on torchvision, mmcv, pytorchcv and OpenVINO Model Zoo (OMZ). Moreover, OpenVINO™ Training Extensions supports auto-configuration functionality to choose a suitable model template based on the dataset. We will further extend our functionality to make training as much simple as possible for obtaining accurate, fast and light models ready to integrate into your projects.
```
Comment (chuneuny-emily) [inline docs/source/guide/get_started/introduction.rst]: Can you change OpenVINO™ Training eXtensions (OpenVINO™ Training Extensions) -> OpenVINO™ Training Extensions ?

File: docs/source/guide/tutorials/base/deploy.rst [hunk 3]
```diff
@@ -37,7 +37,7 @@ But it can be runned for any task in the same manner.
                --load-weights outputs/openvino/openvino.xml
                --save-model-to outputs/deploy
 
-    2023-01-20 09:30:40,938 | INFO : Loading OpenVINO OTXDetectionTask
+    2023-01-20 09:30:40,938 | INFO : Loading OpenVINO OpenVINO™ Training ExtensionsDetectionTask
```
Comment (chuneuny-emily) [inline docs/source/guide/tutorials/base/deploy.rst]: Can we keep original one? This is log info, so looks like it's okay.

File: docs/source/guide/tutorials/base/how_to_train/detection.rst [hunk 4]
```diff
@@ -369,7 +369,7 @@ To learn more about optimization, refer to `NNCF repository <https://github.com/
 
   ...
 
-  2023-01-10 06:29:46,751 | INFO : Loading OpenVINO OTXDetectionTask
+  2023-01-10 06:29:46,751 | INFO : Loading OpenVINO OpenVINO™ Training ExtensionsDetectionTask
```
Comment (chuneuny-emily) [inline docs/source/guide/tutorials/base/how_to_train/detection.rst]: Let's keep original log info.

File: docs/source/guide/get_started/quick_start_guide/cli_commands.rst [hunk 5]
```diff
@@ -206,7 +206,7 @@ Example of the command line to start object detection training:
 
 
 
-It is also possible to start training by omitting the template and just passing the paths to dataset roots, then the :doc:`auto-configuration <../../explanation/additional_features/auto_configuration>` will be enabled. Based on the dataset OTX will choose the task type and template with the best accuracy/speed trade-off.
+It is also possible to start training by omitting the template and just passing the paths to dataset roots, then the :doc:`auto-configuration <../../explanation/additional_features/auto_configuration>` will be enabled. Based on the dataset OpenVINO™ Training Extensions will choose the task type and template with the best accuracy/speed trade-off.
```
Comment (chuneuny-emily) [inline docs/source/guide/get_started/quick_start_guide/cli_commands.rst]: Can you add ,? Based on the dataset OpenVINO™ Training Extensions -> Based on the dataset, OpenVINO™ Training Extensions

File: docs/source/guide/explanation/algorithms/segmentation/instance_segmentation.rst [hunk 6]
```diff
@@ -39,8 +39,8 @@ For the supervised training we use the following algorithms components:
 Dataset Format
 **************
 
-For the dataset handling inside OTX, we use `Dataset Management Framework (Datumaro) <https://github.com/openvinotoolkit/datumaro>`_. For instance segmentation we support `COCO <https://cocodataset.org/#format-data>`_ and `Pascal-VOC <https://openvinotoolkit.github.io/datumaro/docs/formats/pascal_voc/>`_ dataset formats.
-If you have your dataset in those formats, then you can simply run OTX using one line of code:
+For the dataset handling inside OpenVINO™ Training Extensions, we use `Dataset Management Framework (Datumaro) <https://github.com/openvinotoolkit/datumaro>`_. For instance segmentation we support `COCO <https://cocodataset.org/#format-data>`_ and `Pascal-VOC <https://openvinotoolkit.github.io/datumaro/docs/formats/pascal_voc/>`_ dataset formats.
+If you have your dataset in those formats, then you can simply run OpenVINO™ Training Extensions using one line of code:
```
Comment (chuneuny-emily) [inline docs/source/guide/explanation/algorithms/segmentation/instance_segmentation.rst]: OpenVINO™ Training Extensions -> OTX  is better?

File: docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst [hunk 7]
```diff
@@ -40,7 +40,7 @@ For the supervised training we use the following algorithms components:
 Dataset Format
 **************
 
-For the dataset handling inside OTX we use `Dataset Management Framework (Datumaro) <https://github.com/openvinotoolkit/datumaro>`_.
+For the dataset handling inside OpenVINO™ Training Extensions we use `Dataset Management Framework (Datumaro) <https://github.com/openvinotoolkit/datumaro>`_.
```
Comment (chuneuny-emily) [inline docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst]: Can you add ,? For the dataset handling inside OpenVINO™ Training Extensions, we use

File: docs/source/guide/explanation/algorithms/segmentation/instance_segmentation.rst (hunk 6 above)
Comment (supersoob) [reply docs/source/guide/explanation/algorithms/segmentation/instance_segmentation.rst]: it is better to keep as 'then you can simply run using one line of code' not mentioning OTX. Isn't it?

Comment (chuneuny-emily) [reply docs/source/guide/explanation/algorithms/segmentation/instance_segmentation.rst]: I ment,
before: then you can simply run OpenVINO™ Training Extensions using one line of code:
after: then you can simply run OTX using one line of code:

Comment (supersoob) [reply docs/source/guide/explanation/algorithms/segmentation/instance_segmentation.rst]: It's explanation in doc. Is it okay?

**Review by chuneuny-emily — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]
Thank you for the quick fix!

**Pull request merged**